app/views/diabetes-content/health-problems-caused-by-diabetes-type-2.html

```diff
@@ -9,69 +9,89 @@
+
+[Book your annual diabetes review](http://nhsalpha.herokuapp.com/)
+
+If you’re already being treated for high cholesterol and high blood pressure, it’s important you keep taking your medication.  
 
-##What you can do##
+Diabetes also worsens the effects of smoking on your heart.  [Get help to quit smoking](http://www.nhs.uk/LiveWell/Smoking/Pages/stopsmokingnewhome.aspx). 
 
-Managing your blood sugar levels and having regular diabetes check-ups is the best way to lower your risk of complications. 
+##Loss of feeling##
 
-###Checking your heart###
+You should let your GP or diabetes nurse know if you notice any changes in your body. 
 
-You should have your blood fats (cholesterol) and blood pressure checked at least once a year. These checks happen as part of your [annual diabetes check-up]. Diabetes increases your risk of developing heart disease and stroke so it’s important high blood pressure and high cholesterol is spotted and treated early. 
+Diabetes can damage your nerves (neuropathy). This can affect any part of your body, causing:
 
-Diabetes also worsens the harmful effects of smoking on your heart.  [Get help to quit smoking] (http://www.nhs.uk/LiveWell/Smoking/Pages/stopsmokingnewhome.aspx). 
+- numbness
+- pain or tingling
+- problems with sex
+- constipation or diarrhoea
 
-###Watching your feet###
+Early treatment and good blood sugar control can prevent nerve damage getting worse. 
 
-You should look after your feet and check them every day when you have diabetes.  
+##Looking after your feet##
 
-Diabetes can reduce the blood supply to your feet cause a loss of feeling.  This means foot injuries don’t heal well and you may not notice if your foot is sore or injured. 
+Because of your diabetes, you should check your feet every day.  
 
-Simple things are important like:
-keeping feet clean and dry to avoid infection
-trying not to go barefoot outside to avoid niks and cuts
+Diabetes can reduce the blood supply to your feet and cause a loss of feeling.  This means:
 
-Speak to your GP or care team if you notice any changes, including:
+- foot injuries don’t heal well
+- you may not notice if your foot is sore or injured 
+
+Simple things are important, like:
+
+- keeping feet clean and dry to avoid infection
+- trying not to go barefoot outside to avoid niks and cuts
+- wearing shoes that fit well 
+
+Speak to your GP or care team if you notice any changes in your feet, including:
 
 - cuts, cracks or blisters
-- pain
-- itching
-- Numbness of the feet-
+- pain or tingling
+- numb feet
 
 Diabetes UK has advice on [how to check your feet](https://www.diabetes.org.uk/Guide-to-diabetes/Complications/Feet/)
 
-Your feet should also be checked by a foot specialist (podiatrist) every year.
+Your feet should also be checked every year by a foot specialist (podiatrist) or at your annual diabetes review.  
+
+[Book your diabetes foot check](http://nhsalpha.herokuapp.com/booking-with-context/going-for-regular-diabetes-check-ups)
```

Comment:
The link here should be `/book-an-appointment/diabetes-foot-check/find-new-appointment` rather than `http://nhsalpha.herokuapp.com/booking-with-context/going-for-regular-diabetes-check-ups`